assets/styles/main.scss

```diff
@@ -450,7 +450,24 @@ $logo-width: 130px;
 }
 
 .footer-logos {
-  padding: 2rem 0 2rem;
+  padding-top: 1rem;
+}
+
+.footer-content {
+  padding: 0 2rem 0 2rem;
+}
+
+.footer-column {
+  text-align: center;
+  width: 100%;
+
+    @include at-media('desktop') {
+      width: auto;        
+    }
```

Comment:
This is not necessary as you already have a media query below which having the same affect.